packages/ui-extensions/src/surfaces/admin/components/AdminBlock/AdminBlock.ts

```diff
@@ -9,9 +9,14 @@ export interface AdminBlockProps {
   /**
    * The summary to display when the app block is collapsed.
    *
-   * @deprecated App blocks no longer have a collapsed state, so this prop is no longer supported.
+   * @deprecated This prop is no longer supported. Use `collapsedSummary` instead.
    */
   summary?: string;
```

Comment:
Can you clarify why we want to introduce a new prop instead of just re-using the existing one?

Reply:
We dont expose summary anywhere public in public docs nor do we support collapsing of blocks currently, so this is not a "remove then add", for developers this is just an "add". 

The new name makes it much more clear when the value of this prop will be displayed.